structuralalignment/superposition/theseus.py

```diff
@@ -240,7 +222,12 @@ def _run_theseus(self):
     import sys
     import atomium
 
-    pdb_ids = sys.argv[1:]
+    # pdb_ids = sys.argv[1:3]
+    pdb_ids = ["6HG4", "6HG9"]
+
     models = [atomium.fetch(pdb_id) for pdb_id in pdb_ids]
```

Comment:
If I recall correctly, I think we have to pass the `atomium.Model` not the `atomium.File`? 

```suggestion
    models = [atomium.fetch(pdb_id).model() for pdb_id in pdb_ids]
```